release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,30 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 16 October 2024 - Enhancement - Web apps - Colors web apps adjusted [ID 41100]
+
+​The dataminer.services web apps have been updated to make use of new colors, which will provide better contrast and make the UI easier to read.
+
+#### 16 October 2024 - Enhancement - Account linking & Catalog - 'Logout' changed to 'Sign out' [ID 41024]
+
+In the account linking and Catalog UI, the term "Logout" has been replaced with "Sign out".
+
+#### 16 October 2024 - Fix - Catalog - Deploy trial licensed issues [ID 41024]
+
+Deploy (trial) button in the header on the catalog details page no longer mentions not licensed issue.
```

Comment:
Could you rephrase this description so it's a clear description of the issue users may have encountered (e.g. "When you clicked the *Deploy* or *Deploy trial* button in the Catalog, it could occur that [issue description]")? This way, the way we describe fixes remains consistent, and users who search for the issue will be able to find it more easily.